### ISSUE TITLE
Postgres types are not recognized?

### ISSUE THREAD

**Issue by pyros2097** [vaffel/sql-to-graphql#38]
This is my column data
```js
{ tableName: 'users',
  columnName: 'id',
  ordinalPosition: 1,
  isNullable: 'NO',
  dataType: 'character varying',
  udtName: 'varchar',
  columnKey: 'PRI',
  columnType: null }
```
And this is the error I get
```
nhandled rejection Error: Type "character varying" not recognized
    at getType (/home/playlyfer-4/.nvm/v0.10.40/lib/node_modules/sql-to-graphql/steps/column-to-object.js:127:19)
```

**Comment by rexxars** [comment]
Thanks for the report. Postgres adapter has a few issues right now, indeed. I'll see if I can look into it later today.

**Comment by pyros2097** [comment]
Man That was fast..

**Comment by pyros2097** [comment]
Fixed the character varying type but then,
Ok I use custom domains for column types like
```SQL
CREATE DOMAIN Name      VARCHAR(100)   NOT NULL;
```
This might be harder than I thought
```js
{ tableName: 'users',
  columnName: 'name',
  ordinalPosition: 2,
  isNullable: 'YES',
  dataType: 'name',
  udtName: 'name',
  columnKey: null,
  columnType: null }
Unhandled rejection Error: Type "name" not recognized
```
needs a little research..

**Comment by heri** [comment]
I was looking to use this but "json" type is also not supported

**Comment by tjwebb** [comment]
@heri the authors wrote thousands and thousands of lines of code for this project and published it for you for free. Maybe you can take some time to add support for the json type in postgres.

**Comment by heri** [comment]
Hi @tjwebb I'll look into it and see what I can do -- however I have already transformed my db into graphql by hand.

**Comment by sheki** [comment]
This fails with, "timestamp with time zone" I am assuming the postgres support is not great.

**Comment by rexxars** [comment]
@sheki This is true. I don't have any personal projects using Postgres, so there are probably several issues related to Postgres.

I'd be willing to take a look at a basic fix for data types if someone can provide me with:
* A simple schema with the data type
* A few rows of sample data for it
* Some thoughts on the preferred way of exposing this through GraphQL - whether it be a custom data type, cooerce to a primitive etc.

**Comment by daniellmorris** [comment]
One more type I would recommend for postgreSQL is to add the type 'timestamp without time zone'. I fixed it locally on my system by adding the line 
`case 'timestamp without time zone':`
Just after the line
`case 'timestamp with time zone':`
in steps/column-to-object.js

**Comment by shaunc** [comment]
NB -- citext also causes a problem:

    $ sql2graphql ...
    SELECT enum_range(NULL::citext) AS citext - function enum_range(citext) does not exist
